builtin/logical/pki/path_config_acme.go

```diff
@@ -263,6 +263,14 @@ func (b *backend) pathAcmeWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	// Check to make sure that we have a proper value for the cluster path which ACME requires
+	if config.Enabled {
+		_, err = getBasePathFromClusterConfig(sc)
```

Comment:
Should we warn on the read API perhaps?

Reply:
You mean if ACME is enabled and path is not set?

Reply:
Right, since I think this read operation happens on all clusters rather than just the primary, so if an operator reads its config endpoint and sees the warning, they could set it.

Reply:
Cool implemented within https://github.com/hashicorp/vault/pull/20838/commits/8ebfd057f70623d893f3805bd346128213cef834